tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -565,6 +579,202 @@ bool FindConv2DWithBatchNormAndActivation(
   return true;
 }
 
+bool FindPadWithConv2D(const RemapperContext& ctx, int node_index,
+                       PadWithConv2D* matched) {
+  const auto* conv_node_view = ctx.graph_view.GetNode(node_index);
+  const auto* conv_node_def = conv_node_view->node();
+  // Root of the pattern must be a Conv2D.
+  if (!IsConv2D(*conv_node_def)) return false;
+
+  // Check that Conv2D is in inference mode.
```

Comment:
Conv doesn't have an is_training mode, does it? (unlike batch_norm)

Reply:
True! Bad copy pasta :D